content/dharma/abstract-data-structures.md

```diff
@@ -0,0 +1,118 @@
+---
```

Comment:
@Goku-kun Make sure that the parent folder name for this article is the same as your Codecademy username in all lowercase (emphasis mine):

~content/_**dharma**_/abstract-data-structures~ -> content/_**goku-kun**_/abstract-data-structures

Also, the name of the markdown file (currently `abstract-data-structures.md`) must be the same as the `Title` (lowercase and delimited with dashes if needed), which is different.

~`abstract-data-structures.md`~ -> `introduction-to-adts-in-javascript.md`